test/unit/externals.test.ts

```diff
@@ -0,0 +1,85 @@
+import { expect, describe, it } from "vitest";
+import { applyProductionCondition } from "../../src/rollup/plugins/externals";
+
+describe("externals:applyProductionCondition", () => {
+  const applyProductionConditionCases = [
+    {
+      name: "vue-router@4.1.6",
+      in: {
+        ".": {
+          types: "./dist/vue-router.d.ts",
+          node: {
+            import: {
+              production: "./dist/vue-router.node.mjs",
+              development: "./dist/vue-router.node.mjs",
+              default: "./dist/vue-router.node.mjs",
```

Comment:
```suggestion
              production: "./dist/vue-router.node.mjs",
              development: "./dist/vue-router.dev.node.mjs",
              default: "./dist/vue-router.dev.node.mjs",
```

Reply:
I took it from here https://unpkg.com/browse/vue-router@4.1.6/package.json it doesn't have dev however would be nice if it had. Are you suggesting this for test clarity?

Reply:
Exactly. Otherwise it doesn't show a difference between in and out.